app/jobs/build_submission_bundle_job.rb

```diff
@@ -1,16 +1,17 @@
 class BuildSubmissionBundleJob < ApplicationJob
-  def perform(submission)
+  def perform(submission_id)
```

Comment:
I don't know about the logging part, but my understanding is when you enqueue a job with an ActiveRecord model as a parameter these days it enqueues it by its `to_global_id` representation, which is basically just the app name, class name, and id with some slashes.

```
irb(main):002:0> Intake.last.to_global_id.to_s
  Intake Load (1.2ms)  SELECT "intakes".* FROM "intakes" ORDER BY "intakes"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> "gid://vita-min/Intake::CtcIntake/58"
```

So it's just a quick way to avoid having to `Submission.find(...)` in the job and shouldn't have those downsides.

Can you show me where any filter-worthy params might be showing up in the logs? This is what I see:

```
09:21:38 worker.1  | [Worker(host:Traviss-MBP pid:21576)] Job BuildSubmissionBundleJob [35d38cff-4244-4d20-8e6e-7374d37e8549] from DelayedJob(default) with arguments: [{"_aj_globalid"=>"gid://vita-min/EfileSubmission/1"}] (id=40) (queue=default) RUNNING
```

Reply:
I remember being scared away from passing models as arguments in the past for reasons like you describe, but as far as I can tell Rails has been doing the globalid thing since Rails 4.2 which dates back to 2014. I think Ben Sheldon got me in the habit of moving beyond the past and just passing the models without worry.

I saw the eager loading part; I'm not sure it makes enough of a difference in the case of a single item, since it should only effect whether the extra queries happen immediately or 5ms later when they're needed.